src/Admin/HostedServices/AzureQueueBlockIpHostedService.cs

```diff
@@ -35,7 +36,7 @@ protected override async Task ExecuteAsync(CancellationToken cancellationToken)
                     {
                         try
                         {
-                            await BlockIpAsync(message.MessageText, cancellationToken);
+                            await BlockIpAsync(message.DecodeMessageText(), cancellationToken);
```

Comment:
The service that queues these messages (`AzureQueueBlockIpService`) isn't backed by `AzureQueueService` so will not be sending encoded messages.

Currently `AzureQueueBlockIpService` just delivers ip addresses, so it'll constantly fail base64 decode operations.

Reply:
Ah, when it checked references I could have sworn that came up, but it was when I was keying on `MessageText` itself so my wires got crossed, thanks Matt! Updated.